sa/_db-next/migrations/20170731115209_AddOrders.sql

```diff
@@ -0,0 +1,25 @@
+
+-- +goose Up
+-- SQL in section 'Up' is executed when this migration is applied
+CREATE TABLE orders (
+       id bigint(20) NOT NULL AUTO_INCREMENT,
+       registrationID bigint(20) NOT NULL,
+       expires datetime NOT NULL,
+       csr mediumblob NOT NULL,
+       error blob DEFAULT NULL,
+       certificateSerial varchar(255) DEFAULT NULL,
+       status varchar(255) NOT NULL,
+       PRIMARY KEY(id),
+       KEY reg_expires (registrationID, expires)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+CREATE TABLE orderToAuthz (
+       orderID bigint(20) NOT NULL,
+       authzID varchar(255) NOT NULL,
+       PRIMARY KEY order_authz (orderID, authzID)
```

Comment:
I think we actually need to separate keys: one by `orderID` and one by `authzID`. We need to able to look up orders given an authz, and authzs given an order. For lookups, we don't need an index that combines both, but for the sake of having a unique primary key, it probably makes sense to have one. So I'd suggest keeping the current PRIMARY KEY and adding another `KEY authzID (authzID)`